src/resources/postcss/views/month/_day.pcss

```diff
@@ -73,4 +73,13 @@
 	.tribe-events-calendar-month__day-date--disabled {
 		color: var(--color-text-disabled);
 	}
+
+	.tribe-events-calendar-month__day-events {
```

Comment:
we should put these display styles on the entire calendar day rather than here, since we have a button for mobile.

Reply:
The idea of this one was also to wrap the regular events and add the required side padding (no padding or anything for the multiday events).

Reply:
And also, wrapping the entire day will make us re-work on the whole multiday structure as it won't work the way it is.

Reply:
the padding is fine, i mean the `display: none` and `display: block`. see https://github.com/moderntribe/the-events-calendar/pull/2544#discussion_r291478904